aws/node/main.tf

```diff
@@ -90,7 +90,7 @@ output "public_ip" {
   value = ["${aws_instance.instance.*.public_ip}"]
 }
 output "hostnames" {
-  value = ["${aws_instance.instance.*.tags.Name}"]
+  value = ["${split(" ", replace(join(" ",formatlist("ip-%s", aws_instance.instance.*.private_ip)),".","-"))}"]
```

Comment:
this mode shouldn't belong to Cloudflare, but it's fine as long as you write it in the description of the PR.